lib/Terms/ChangeLanguage.php

```diff
@@ -93,4 +100,96 @@ public function change_language( int $term_id ) : void {
 		remove_filter( 'ubb_use_post_lang_filter', '__return_false' );
 		remove_filter( 'ubb_use_term_lang_filter', '__return_false' );
 	}
+
+	/**
+	 * Handle terms with duplicate slugs when language is changed.
+	 *
+	 * @since Unreleased
+	 *
+	 * @param int $term_id
+	 * @param string $lang
+	 * @param string|null $old_lang
+	 * @return void
+	 */
+	public function duplicate_term_language_change( int $term_id, string $lang, ?string $old_lang ) : void {
```

Comment:
I think we probably don't need this. If we change the language on our side and then force an update of the term (this could also be valid for a post) we should get an updated slug without needing to search on our end.

Either update with nothing or with the data that should be unique.

Reply:
Forcing an update via the `wp_update_term` does not seem to work due to the duplicate check.
![image](https://github.com/user-attachments/assets/2a064ad4-19bf-444c-9bd4-f9644ef19d77)
If the term being updated has the lowest id of the duplicates, it will be the one found in the duplicate check and the term slug will not be made unique.

Reply:
To bad 😞. Maybe WordPress will eventually change this and we might optimize in the future.